sources/packages/api/src-sql/ProgramYear/Create-initial-program-year.sql

```diff
@@ -0,0 +1,9 @@
+-- Inserting initial values for 'program_year' table. This will be part of table creation migration. So, no drop migration sql, this data will destroy with table in drop migration
+INSERT INTO program_years(program_year,program_year_desc, form_name)
+VALUES('2021-2022','Study starting between August 01, 2021 and July 31, 2022', 'SFAA2021-22');
```

Comment:
We will need to provide Camunda a program year "code", like "2021-2022", to be used on DMN, right?
So, I believe that we need keep the format in sync like "2021-2022" vs "2021/22". Does it make sense?